actionpack/lib/action_controller/metal/helpers.rb

```diff
@@ -12,6 +12,12 @@ module ActionController
   # In previous versions of \Rails the controller will include a helper which
   # matches the name of the controller, e.g., <tt>MyController</tt> will automatically
   # include <tt>MyHelper</tt>. To return old behavior set +config.action_controller.include_all_helpers+ to +false+.
+  # You can include the aforementioned in any environment file you wish (e.g. config/environments/production.rb) or alternatively
```

Comment:
Changing this behaviour for a specific environment doesn't sound sensible to me - unless you can think of a reason that would be useful, I think we should just recommend setting it in `config/application.rb`.

Reply:
ok.